.ci.yaml

```diff
@@ -188,13 +188,13 @@ targets:
       caches: >-
         [
           {"name":"gradle","path":"gradle"},
-          {"name": "openjdk", "path": "java"}
+          {"name": "openjdk", "path": "java11"}
```

Comment:
Name is the one that needs to change, path should be kept as java.

Reply:
there are many caches in this file that have this entry: `{"name": "openjdk", "path": "java11"}`. Are those set incorrectly?

Reply:
Yes the are all incorrect, the modules will add cache/java to the path and this cache won't have any effect.

Reply:
got it. can the name be any arbitrary string?

Reply:
we are usually using dependency_<version> for named caches.

Reply:
Done